html/pages/index.html

```diff
@@ -1,4 +1,137 @@
----
-layout: default
-title: Home
----
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <!-- <meta content="" name=""> -->
+    <!-- <link href="" rel="stylesheet" type="text/css"> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+  </head>
+
+  <body>
+    <div class="wrapper">
+
+      <header>
+        <h1 class="logo">Movies&amp;Stuff</h1>
+        <nav class="navigation">
+          <ul>
+            <li> <a href="#movies"> Movies </a> </li>
+            <li> <a href="#tv-shows"> TV Shows </a> </li>
+            <li> <a href="#celebrities"> Celebs &amp; Photos</a> </li>
+            <li> <a href="#news"> News </a> </li>
+          </ul>
+        </nav>
+      </header>
+
+      <section class="suggested">
+          <!-- It seems that the following element must be a link to a video. It's just a plain image for now.
+          Background image will be set in css -->
+
+        <img src="{{root}}images/deadpool-preview.jpg" alt="deadpool-preview">
+        <!-- this table wraps typical data for suggested movies -->
+        <table class="suggested-table-data">
+          <tbody>
+            <tr>
+              <td class="suggested-title" colspan="2">Deadpool</td>
+              <td class="suggested-rating" colspan="2">
+                <img src="{{root}}images/rating-star.png" alt="a rating star-1">
+                <img src="{{root}}images/rating-star.png" alt="a rating star-2">
+                <img src="{{root}}images/rating-star.png" alt="a rating star-3">
+                <img src="{{root}}images/rating-star.png" alt="a rating star-4">
+                <img src="{{root}}images/rating-star.png" alt="a rating star-5">
+              </td>
+            </tr>
+            <tr >
+              <td class="suggested-description" colspan="4">A former Special Forces operative turned mercenary is subjected to a rogue experiment that leaves him with accelerated healing powers, adopting he alter ego Deadpool.</td>
+            </tr>
+            <tr>
+              <td class="suggested-mpaa-rating">R</td>
+              <td class="suggested-duration">108 min</td>
+              <td class="suggested-genre">Action Adeventure, Comedy</td>
+                <!-- should I actually use time tag? -->
+              <td class="suggested-release-date"> <time datetime="2016-02-12">12 February 2016</time></td>
+            </tr>
+        </table>
+      </section>
+
+      <!-- A section with suggested popular movies
+            id attribute used for anchoring
+      -->
+      <section id="movies" class="popular-movies">
+
+        <h2>Popular Movies</h2>
+        <a href="">View All Movies</a>
+          <!-- each pic is supposed to be a link to more detailed info, isn't it? -->
+        <a href="">
+          <img src="{{root}}images/the-martian-preview.jpg" alt="the-martian-preview">
```

Comment:
The `alt` text should be human-readable. It's for humans. Also, I think we need `title` attribute.